utils/version.py

```diff
@@ -3,13 +3,17 @@
 from utils.console import print_step
 
 
-def checkversion(__VERSION__):
+def checkversion(__VERSION__:str):
     response = requests.get("https://api.github.com/repos/elebumm/RedditVideoMakerBot/releases/latest")
     latestversion = response.json()["tag_name"]
     if __VERSION__ == latestversion:
         print_step(f"You are using the newest version ({__VERSION__}) of the bot")
         return True
+    #FOR ERROR HANDLING, SO VERSION CAN EASILY BE DETECTED

```

Comment:
Now there isn't a notification that you're using an older version. I do like the idea of showing the beta version message. Maybe do it with a higher or lower version if statement?